src/test/java/com/amalitech/gpuconfigurator/repository/UserRepositoryTest.java

```diff
@@ -0,0 +1,7 @@
+package com.amalitech.gpuconfigurator.repository;
```

Comment:
Since your repository is just implementing JPARepository, you don't necessarily need a test file for your repository. Remove file